src/integration-test/java/com/commercetools/sync/integration/externalsource/shoppinglists/ShoppingListSyncIT.java

```diff
@@ -110,202 +119,173 @@ void sync_WithSameShoppingList_ShouldNotUpdateCustomer() {
-                        .key("dummy-key-2")
-                        .customer(prepareCustomer().toResourceIdentifier())
-                        .description(LocalizedString.ofEnglish("new-shoppinglist-description"))
-                        .build();
-
-        final ShoppingListSyncStatistics shoppingListSyncStatistics = shoppingListSync
-                .sync(singletonList(newShoppingListDraft))
-                .toCompletableFuture()
-                .join();
-
-        assertThat(errorMessages).isEmpty();
-        assertThat(warningMessages).isEmpty();
-        assertThat(exceptions).isEmpty();
-        assertThat(updateActionList).isEmpty();
-
-        assertThat(shoppingListSyncStatistics).hasValues(1, 1, 0, 0);
-
-    }
-
-    @Disabled
-    @Test
-    void sync_WithNewShoppingListPlusLineItems_ShouldCreateShoppingList() {
-
-        final ShoppingListDraft newShoppingListDraft =
-                ShoppingListDraftBuilder.of(LocalizedString.ofEnglish("dummy-name-2"))
-                        .key("dummy-key-2")
-                        .lineItems(prepareLineItemDrafts())
-                        .textLineItems(prepareTextLineItemDrafts())
-                        .description(LocalizedString.ofEnglish("new-shoppinglist-description"))
-                        .build();
-
-        final ShoppingListSyncStatistics shoppingListSyncStatistics = shoppingListSync
-                .sync(singletonList(newShoppingListDraft))
-                .toCompletableFuture()
-                .join();
-
-        assertThat(errorMessages).isEmpty();
-        assertThat(warningMessages).isEmpty();
-        assertThat(exceptions).isEmpty();
-        assertThat(updateActionList).isEmpty();
-
-        assertThat(shoppingListSyncStatistics).hasValues(1, 1, 0, 0);
-
-    }
-
-    @Disabled
-    @Test
-    void sync_WithNewShoppingListPlusTextLineItems_ShouldCreateShoppingList() {
-
-        final ShoppingListDraft newShoppingListDraft =
-                ShoppingListDraftBuilder.of(LocalizedString.ofEnglish("dummy-name-2"))
-                        .key("dummy-key-2")
-                        .textLineItems(prepareTextLineItemDrafts())
-                        .description(LocalizedString.ofEnglish("new-shoppinglist-description"))
-                        .build();
-
-        final ShoppingListSyncStatistics shoppingListSyncStatistics = shoppingListSync
-                .sync(singletonList(newShoppingListDraft))
-                .toCompletableFuture()
-                .join();
-
-        assertThat(errorMessages).isEmpty();
-        assertThat(warningMessages).isEmpty();
-        assertThat(exceptions).isEmpty();
-        assertThat(updateActionList).isEmpty();
-
-        assertThat(shoppingListSyncStatistics).hasValues(1, 1, 0, 0);
-
+                + "(0 created, 0 updated and 0 failed to sync).");
     }
 
     @Test
     void sync_WithModifiedShoppingList_ShouldUpdateShoppingList() {
```

Comment:
Please add a comment or link to the example in ADR that this test is trying to address.

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/626/commits/d722e5dbb9cb5c557ce034b79bcafc71c7c58428

Reply:
I was thinking more like a compare where you have old and new shopping lists, so I can see the diff:

```
oldShoppingList{
name: "abc"
[LineItem1, LineItem2]
}

newShoppingList{
name: "newAbc"
[LineItem1]
}
```

Reply:
would be better, but in the end, I am thinking the scenarios are clear.